tests/conftest.py

```diff
@@ -319,7 +319,7 @@ def mock_env(monkeypatch, request):
             yield
 
 
-@pytest_asyncio.fixture()
+@pytest_asyncio.fixture(scope="function")
```

Comment:
There was a recent change to the asyncio plugin that requires explicit scopes for async fixtures